js/ui/menu/ui.menu.js

```diff
@@ -541,14 +541,16 @@ class Menu extends MenuBase {
         return submenu;
     }
 
+    _getKeyboardListeners() {
```

Comment:
FYI
This change makes this regression, it worked correctly before:
[T920355 - Menu - Item in onItemClick handler is incorrect when using keyboard navigation to "click" items](https://supportcenter.devexpress.com/internal/ticket/details/T920355)